stable/postgresql/values.yaml

```diff
@@ -57,6 +57,23 @@ volumePermissions:
   securityContext:
     runAsUser: 0
 
+##
+## Init container can be separately added to Master and Slave stateful set by changing respectively
+## initContainer.masterEnabled and initContainer.slavesEnabled values
+##
+
```

Comment:
Just to be consistent with the rest of the comments, this line is not needed, you can add the variables just after the `##`

Reply:
Removed.